forecast_tools.R

```diff
@@ -86,8 +86,8 @@ plot_data = function(data) {
     text = getURL(
       "https://raw.githubusercontent.com/weecology/PortalData/master/Rodents/moon_dates.csv"))
   target_moon=unique(data$NewMoonNumber)
-  period_code = dplyr::filter(newmoons_table, newmoons_table$NewMoonNumber == target_moon) %>%
-    dplyr::select(Period)
+  period_code = as.integer(dplyr::filter(newmoons_table, newmoons_table$NewMoonNumber == target_moon) %>%
+    dplyr::select(Period))
```

Comment:
OK, great. To update the PR do the following:

1. Change the code in your branch
2. `git add forecast_tools.R`
3. `git commit --amend`
4. `git push repair_plot_data_func`

Let me know if you run into any trouble and add a short comment to this issue letting me know it's done if everything works so that I get it merged.